### ISSUE TITLE
Missing feedback for  unverified sessions

### ISSUE THREAD

**Issue by gedeondoescode** [commetchat/commet#101]
**Describe the bug**
There's no feedback letting the user know that the encrypted messages cannot be decrypted.

**To Reproduce**
1. Sign in for the first time
2. Look in unverified room
3. No messages appear (no feedback suggesting an unverified session)

**Expected behavior**
An error stating that the user has not verified their session to decrypt the message

<details open>
<summary>Device Information</summary>
<br>

**Device**
Platform: `linux`
Version: `v0.1.0`
Git Hash: `9f69a41ff2a5a2fb9d7cd9036197c2599c6cf9ca`
Detail: `linux-x64`

**System Info**
Name: `Pop!_OS`
Version: `22.04 LTS`

</details>

**Comment by Airyzz** [comment]
I think this is kind of two issues, as we currently dont have any indicator for unverified sessions, and we also arent displaying anything for events which failed to decrypt.

Lets keep this issue around until we have indicators for session verification

As for indicating encryption failures in messages, thats being added over in #106

**Comment by freddy1301** [comment]
Its been a Year now. Where is our Session Verification!?

I just discovered Commet and Matrix as a whole and Commet is really close to be my favorite Client (Threads would be nice). But without being able to Verify your Session and unencrypt previous messages and on top of that having a Warning next to your user that shows for everyone else using different Clients, its unusable in my Opinion. 

Other than that, well done! Really great end result so far. Maybe Ill donate a few €

**Comment by Airyzz** [comment]
> Its been a Year now. Where is our Session Verification!?

sorry, I am the only one working consistently on this project, I cannot get everything done in a timely manner. that having been said, it has been possible to verify your session since the very first version. If it is not working for you, that is a separate issue.

> Threads would be nice. 
threads are supported in the latest pre-release builds

**Comment by maurges** [comment]
Is there a way to verify my session at all? I didn't find any buttons for it, and nothing invited me to when I tried to initiate the session verification from my other device

**Comment by theMimolet** [comment]
Yeah, not being able to verify basically makes it unusable, all my DMs are encrypted 🥲

**Comment by BerryCutePixels** [comment]
having the same issues here and verify being basically unuseable. verify only kind of works in settings < security, but its not verifying correctly for encrypted servers and dms. Had to open up element to do it there and then it works - which defeats the purpose of using this program
